user_docs/en/userGuide.t2t

```diff
@@ -1997,6 +1997,35 @@ By default only three modes are included.
 Note that it is necessary to check at least one mode.
 This option is not available if you have started NVDA with [WASAPI disabled for audio output #WASAPI] in Advanced Settings.
 
+==== Applications volume ====[AppSoundVolume]
+
+This slider allows to set volume of all currently running applications except for NVDA.
+If any new applications start to output sound, their volume will also be affected.
+This slider can also be controlled via the following keyboard commands:
+
+%kc:beginInclude
+|| Name | Key | Description |
+| Increase applications volume | ``NVDA+alt+pageUp`` | Increases volume of all applications except NVDA by 5%. |
+| Decrease applications volume | ``NVDA+alt+pageDown`` | Decreases volume of all applications except NVDA by 5%. |

```

Comment:
I'm not clear on why Cyrille removed the "by five percent" part; I have no opinion on that as I didn't read the code.
But again I suggest "other" for clarity.

I might also point out that an argument could be made that "applications volume" is a plural possessive, and so should be "applications' volume". I.e. it is the volume belonging to those applications. However I didn't suggest that above, so I won't below.

```suggestion
| Decrease other applications volume | ``NVDA+alt+pageDown`` | Decreases volume of all applications other than NVDA by 5%. |
```

Reply:
I have removed the "5%" because I do not think that this level of detail needs to be put in the command description.

When you increase general volume with the volume button on your keyboard, you do not know the step and that does not cause you any problem; thus, this information is not necessary and just makes the description heavier.